src/makeInfinityAdder.js

```diff
@@ -4,7 +4,26 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
-}
+  return function adder(a) {
+    if (a === undefined) {
+      return 0;
+    }
+
+    let sum = a;
+
+    return function inner(b) {
+      if (b === undefined) {
+        const currantSum = sum;
```

Comment:
There is a typo in the variable name `currantSum`. It should be `currentSum` to correctly represent the current sum.